packages/Webkul/Category/src/Models/Category.php

```diff
@@ -66,8 +66,15 @@ public function getImageUrlAttribute()
      */
     public function filterableAttributes()
     {
+
```

Comment:
2 blank lines are added, please remove this,

Reply:
Ok Wil resubmit with your changes.